src/toil/jobStores/fileJobStore.py

```diff
@@ -70,7 +66,7 @@ def __init__(self, path, fanOut=1000):
                            subdirectories
         """
         super(FileJobStore, self).__init__()
-        self.jobStoreDir = absSymPath(path)
+        self.jobStoreDir = os.path.realpath(path)  # doesn't dereference symlinks
```

Comment:
This comment is confusing. `os.path.realpath` replaces symlinks with [the actual path](https://docs.python.org/3/library/os.path.html#os.path.realpath).  Is this the desired behavior?

Reply:
*sigh*  It's not the desired behavior. I'll adjust and include a test for this.  Thanks for the review, really appreciate it.